views/plugin/xyz/xyzdescriptornodeui.html

```diff
@@ -0,0 +1,8 @@
+<div>
+  <span ng-if="nodeUi.show()" class="flex-shrink-0" ng-click="nodeUi.edit()">
+    <i class="fa fa-pencil-square-o fa-fw c-glyph" title="Edit this layer"></i>
```

Comment:
We typically use [this icon](https://fontawesome.com/v5.15/icons/pencil-alt?style=solid) for edit. In FA4 the classes are `fa fa-pencil`, now that we're on FA5 we should start using `fas fa-pencil-alt`.